docker/CHANGELOG.md

```diff
@@ -2,31 +2,62 @@
 
 All notable changes to the fyne-cross docker images will be documented in this file.
 
-Release cycle won't follow the fyne-cross one, so the images will be tagged using the label
-year.month.day along with the latest one.
+## fyne 1.1.x compatible
 
-# Release 20.12.13
+Latest versions available on dockerhub are
+
+- fyneio/fyne-cross:1.1-base
+- fyneio/fyne-cross:1.1-base-llvm
+- fyneio/fyne-cross:1.1-base-freebsd
+- fyneio/fyne-cross:1.1-android
+- fyneio/fyne-cross:1.1-freebsd-amd64
+- fyneio/fyne-cross:1.1-freebsd-arm64
+- fyneio/fyne-cross:1.1-linux-386
+- fyneio/fyne-cross:1.1-linux-arm64
+- fyneio/fyne-cross:1.1-linux-arm
+- fyneio/fyne-cross:1.1-windows
+
+Release cycle won't follow the fyne-cross one, so the images will be tagged and available on dockehub using the label year.month.day along with the version one.
+
+Example: `fyneio/fyne-cross:1.1-base-21.03.17`
+
+## fyne 1.0.x compatible
+
+Latest version available on dockerhub are:
+- fyneio/fyne-cross:base-latest
+- fyneio/fyne-cross:darwin-latest
+- fyneio/fyne-cross:linux-386-latest
+- fyneio/fyne-cross:linux-arm64-latest
+- fyneio/fyne-cross:linux-arm-latest
+- fyneio/fyne-cross:android-latest
+- fyneio/fyne-cross:freebsd-latest
+
+Release cycle won't follow the fyne-cross one, so the images will be tagged and available on dockehub using the label year.month.day along with the version one.
```

Comment:
What does this mean? I'm not sure I quite follow.

Reply:
The release of a new image usually happen more frequently that a fyne-cross one.
The `yy.mm.dd` tag was introduced to decouple fyne-cross and the docker images versioning. 
For example, let's say that after we release fyne-cross v1.1.0 we want to bump the Go version to the next 1.16.x. 
We won't release a new version for fyne-cross since nothing is changed there but only a new image that we'll tag twice with the `latest` and `yy.mm.dd`.
The `latest` tag is used by fyne-cross internally to allow the pull of the new ones (--pull flag).
The `yy.mm.dd` allow to keep track of the image changes and to use old images via the `--image` flag

Reply:
Ah, I see.